docs/plugins/internationalization.md

```diff
@@ -58,8 +58,11 @@ In an app you can both override the default messages, as well as register transl
 +      }),
 +      createTranslationResource({
 +        ref: myPluginTranslationRef,
-+        messages: {
++        translations: {
 +          zh: () => import('./translations/zh'),
++          pt: () => Promise.resolve({ default:  {
```

Comment:
Think that we should either drop this `Promise.resolve` pattern or add an explicit comment saying that this should only be used for tests and not in production as it's not lazily loaded.

Reply:
Yes, I put this here just because it is easy to test in this way, but agree to put a comment in place for that